src/bindings/js/node/src/tensor.cpp

```diff
@@ -132,6 +132,10 @@ Napi::Value TensorWrap::get_data(const Napi::CallbackInfo& info) {
 }
 
 Napi::Value TensorWrap::get_shape(const Napi::CallbackInfo& info) {
+    if (info.Length() > 0) {
+        reportError(info.Env(), "No parameters are allowed for the get_shape method.");
+        return;
```

Comment:
I have a question, in the case of parameters being passed into the method, while returning , shall I return the correct shape of the tensor along with error reporting or do I return a null value with the error?

Reply:
Here function defined with return value with type Napi::Value. It means that function must return value in any execution branches. In case of error, besides throwing js error you have to return some value of type Napi::Value or it's child types (see [Napi docs](https://github.com/nodejs/node-addon-api/blob/main/doc/value.md)). 
Usually in the case of error we return Undefined.